yarn.lock

```diff
@@ -5740,12 +5740,12 @@ babel-jest@^29.7.0:
     slash "^3.0.0"
 
 babel-loader@^8.0.6:
-  version "8.2.2"
-  resolved "https://registry.yarnpkg.com/babel-loader/-/babel-loader-8.2.2.tgz#9363ce84c10c9a40e6c753748e1441b60c8a0b81"
-  integrity sha512-JvTd0/D889PQBtUXJ2PXaKU/pjZDMtHA9V2ecm+eNRmmBCMR09a+fmpGTNwnJtFmFl5Ei7Vy47LjBb+L0wQ99g==
+  version "8.4.1"
```

Comment:
Rather than force `babel-loader` within the yarn.lock, can we catually upgrade it properly within our packages.json? It is present, [here](https://github.com/hashicorp/boundary-ui/blob/main/ui/desktop/package.json#L67), [here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/package.json#L64) and [here](https://github.com/hashicorp/boundary-ui/blob/main/addons/rose/package.json#L57)

Reply:
great catch, didn't realize it was a dev. dep. ill update this.